src/app/accounts/get-csv-for-account.ts

```diff
@@ -8,8 +8,8 @@ export const getCSVForAccount = async (
   if (wallets instanceof Error) return wallets
 
   const csv = new CsvWalletsExport()
-  for (const wallet of wallets) {
-    await csv.addWallet(wallet.id)
-  }
+
+  await csv.addWallet(wallets)
```

Comment:
Just curious, why the change iterating inside the function rather than outside?

I'm fine with the change, but would you mind just renaming it to `addWallets`?